builds/misc/metrics-collector-images-release.yaml

```diff
@@ -0,0 +1,88 @@
+name: $(version)
+trigger: none
+pr: none
+
+variables:
+  NugetSecurityAnalysisWarningLevel: warn
+  Build.Configuration: release
+  
+jobs:
+################################################################################
+  - job: MetricsCollector_Linux
+################################################################################
+    displayName: Metrics Collector Linux Build
+    timeoutInMinutes: 30
+    pool:
+      vmImage: 'ubuntu-18.04'
+    steps:
+      - template: ../templates/install-dotnet2.yaml
```

Comment:
You're right that AzFunc is bumped to 3.1 in `release/1.2 & master` branches. 
Check w/ Anca if the same has been done for `release/1.1`.

Reply:
I mean.. We are dealing with MetricsCollector module here. Hypothetically you won't need it for this task.

Reply:
Correct, but we don't need the .NET Core 2.1 SDK to build the _metrics collector image_, so we don't need `install-dotnet2.yaml` in this YAML file. We only need the .NET Core 3.1 SDK here.

Reply:
It is necessary though because I am building all CSharp resources from the solution file at the root of our repo. Here is why I did it this way:
- We will know the repo is in a good state before releasing.
-  Using same build / publish scripts (and therefore flags and other logic) as the build images pipeline. So no surprises when we release.

Alternatively, I can only build the metrics collector if we prefer that approach?

Reply:
That's fine. We can always refine it later if we want to.
@yophilav The .NET Core 2.1 dependency was removed in 1.1 too, but then the change was reverted from some reason. We should look into when we'll be able to bring it back.